src/Search/Search.test.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { findByTestAttr, checkProps } from '../../Utils';
+import { Search } from './Search';
+
+const setup = (props = {}) => {
+  const component = shallow(<Search {...props} />);
+  return component;
+};
+
+describe('Search component', () => {
+  describe('should render Search', () => {
```

Comment:
you don't need inner `describe` blocks here, no reason for this

Reply:
Removed